### ISSUE TITLE
[INSTALL]: Installation on MAC

### ISSUE THREAD

**Issue by Sidney-Bishop** [pycaret/pycaret#3060]
### Installation check

- [X] I have read the [installation guide](https://pycaret.gitbook.io/docs/get-started/installation).

### Platform

macOS-10.16-x86_64-i386-64bit

### Installation Method

pip install

### pycaret Version

2.3.10

### Python Version

3.8.13

### Description

Hi,

Following the instructions on the PyCaret webpage [here](https://pycaret.gitbook.io/docs/get-started/installation) I've created a conda environment as instructed encountered the following issue when i execute this line of code (by running python in the terminal) which is part of the "Classification" "QuickStart" tutorial code on the same web page.

`from pycaret.classification import *`

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/pycaret/classification.py", line 10, in <module>
    import pycaret.internal.tabular
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/pycaret/internal/tabular.py", line 54, in <module>
    import pycaret.internal.preprocess
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/pycaret/internal/preprocess.py", line 34, in <module>
    from lightgbm import LGBMClassifier as lgbmc
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/lightgbm/__init__.py", line 8, in <module>
    from .basic import Booster, Dataset, Sequence, register_logger
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/lightgbm/basic.py", line 110, in <module>
    _LIB = _load_lib()
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/lightgbm/basic.py", line 101, in _load_lib
    lib = ctypes.cdll.LoadLibrary(lib_path[0])
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/ctypes/__init__.py", line 451, in LoadLibrary
    return self._dlltype(name)
  File "/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/ctypes/__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/lightgbm/lib_lightgbm.so, 0x0006): Library not loaded: '/usr/local/opt/libomp/lib/libomp.dylib'
  Referenced from: '/Users/jasonroche/opt/anaconda3/envs/PyCaret_env/lib/python3.8/site-packages/lightgbm/lib_lightgbm.so'
  Reason: tried: '/usr/local/opt/libomp/lib/libomp.dylib' (no such file), '/usr/local/lib/libomp.dylib' (no such file), '/usr/lib/libomp.dylib' (no such file)

There seems to be an issue with lightgbm. I've tried some of the suggestions on other threads where folks suggest installing lightgbm using brew, but i get the same message as them"

Warning: lightgbm 3.3.3 is already installed and up-to-date.

Despite this I can import lightgbm in python.

So I tried 

`conda install -c conda-forge lightgbm`

But now get this error when I try to import it.

<frozen importlib._bootstrap>:219: RuntimeWarning: scipy._lib.messagestream.MessageStream size changed, may indicate binary incompatibility. Expected 56 from C header, got 64 from PyObject

Any help / ideas for how to get pyCaret woking on a Mac M1 Pro Monterey would be greatly appreciated !

Best Jason
 

### Installation Logs

<details>

Replace this line with the installation logs.

</details>

**Comment by Sidney-Bishop** [comment]
@ngupta23  thanks I didn't check out the FAQ's. I have tried them since but unsuccessfully. I followed the steps of this link (https://pareekshithkatti.medium.com/setting-up-python-for-data-science-on-m1-mac-ced8a0d05911)  but even after manually installing the additional (the those mentioned in the requirements.txt) required libraries I got an incompatible version of scikit-learn. I'm just checking it out so I'll use the docker image.
